CHANGELOG.md

```diff
@@ -1,8 +1,15 @@
+# dbt_pendo_source v0.5.0
+## 🚨 Breaking Changes 🚨:
+- Updated models `stg_pendo__guide_event`, `stg_pendo__poll_event`, and `stg_pendo__event` to add `_fivetran_id`, which was recently added per the Fivetran Pendo Connector's [March 2023 release notes](https://fivetran.com/docs/applications/pendo/changelog#march2023). ([#21](https://github.com/fivetran/dbt_pendo_source/pull/21))
+
```

Comment:
also since there's a parallel release in the transform package we should probably link to the transform changelog entry

Reply:
Agreed and updated.